### PR TITLE
fix/formatting => dev

### PR DESCRIPTION
**Pull request by Mawfyy**
add formatting, testing and linter tasks for pipeline for check and maintain the codebase.

### CODE REVIEW

File: src/scraper/sites/manga/nhentai/Nhentai.ts
Comment (TokyoTF) [inline src/scraper/sites/manga/nhentai/Nhentai.ts:1]: In headers "Connection": "keep-alive" and "Cookie" it is not required

**Review by TokyoTF — APPROVED** [review]
But the rest is fine

File: src/scraper/sites/manga/nhentai/Nhentai.ts
Comment (Mawfyy) [reply src/scraper/sites/manga/nhentai/Nhentai.ts:1]: Yep, with that reason i revert that commit

**Review by barrientosvctor — COMMENTED** [review]
This commit needs to specify an eslint configuration into a [.eslintrc.*](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats) for its correct use. The configuration has to be discussed with the team to use one that seems best to everyone.

**Review by barrientosvctor — APPROVED** [review]
The rest is OK.

**Comment by Mawfyy** [comment]
> This commit needs to specify an eslint configuration into a [.eslintrc.*](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats) file for its correct use. The configuration has to be discussed with the team to use one that seems best to everyone.

Okay, I'll check about .eslintrc today, thank you.

**Review by koikiss-dev — APPROVED** [review]

**Pull request merged**